static_src/components/users_invite.jsx

```diff
@@ -7,7 +7,6 @@
 import React from 'react';
 
 import Action from './action.jsx';
-import ErrorMessage from './error_message.jsx';
```

Comment:
Good question. 

This looks like it does it: https://github.com/depcheck/depcheck